packages/client/lib/rpc/modules/eth.ts

```diff
@@ -957,6 +963,34 @@ export class Eth {
 
     return bufferToHex(tx.hash())
   }
+
+  async getProof(params: [string, string[], string]) {
```

Comment:
```suggestion
  /**
   * Returns an object with data about the sync status or false.
   * @param params An array of three parameters:
   *   1. address of the account
   *   2. array of storage-keys which should be proofed and included
   *   3. integer block number, or the string "latest" or "earliest"
   * @returns An account object
   **/
  async getProof(params: [string, string[], string]) {
```

Reply:
great idea, thanks for writing the typedoc :) it `@returns {@link Proof}` though not simply an account object, which seems to be:

```typescript
type Proof = {
  balance: PrefixedHexString
  codeHash: PrefixedHexString
  nonce: PrefixedHexString
  storageHash: PrefixedHexString
  accountProof: PrefixedHexString[]
  storageProof: StorageProof[]
  address: PrefixedHexString
}
```

I will modify your comment then merge it in along with anything else I might find as I start my review now 👍